src/Sinks/formatlogger.jl

```diff
@@ -1,25 +1,25 @@
 
 struct FormatLogger <: AbstractLogger
-    f::Function
+    formatter
```

Comment:
More descriptive now that there is no type information

Reply:
Do you think its worth making this type parametric,
so we do not trigger a dynamic dispatch every time?

Reply:
We can make this parametric. I only didn't do so as I wanted to ensure this wasn't breaking